packages/mjml-column/src/index.js

```diff
@@ -191,7 +191,7 @@ export default class MjColumn extends BodyComponent {
     return `
       <table
         ${this.htmlAttributes({
-          background: this.getAttribute('background-color'),
+          bgcolor: this.getAttribute('background-color'), // TODO: move this attribute into css style as recommended in https://developer.mozilla.org/cs/docs/Web/HTML/Element/table
```

Comment:
It wasn't here in MJML3 in the first place, can you remove completely this line too please ?

Reply:
Currently it is supported (and it's really useful) - see [mjml-column](https://mjml.io/documentation/#mjml-column), so I'd like to think about how we can preserve this property. Afaik this patch would fix it (as bgcolor is ancient but still supported), right?

Reply:
https://github.com/tkafka/mjml/blob/472c47c845eb0feebe66a5823a8b6479adbf7f02/packages/mjml-column/src/index.js#L60 Background color is already applied here. A lot of logic has been moved from column to section in MJML4 that's why we still see some unwanted attribute

Reply:
@iRyusa But then I'll lose an option to set background of individual columns (which is a feature I am using now, and a reason I found this problem), right? Or would I then do it with CSS class instead?

Reply:
> https://github.com/tkafka/mjml/blob/472c47c845eb0feebe66a5823a8b6479adbf7f02/packages/mjml-column/src/index.js#L60 Background color is already applied here.

As said here, no. `background-color` is applied on the wrapping table already so it's just `bgcolor/background` HTML attributes are just useless and color will be applied through the `table` inline style